examples/computer_vision/mmdetection/README.md

```diff
@@ -0,0 +1,51 @@
+# MMDetection in Determined
+This example allows you to use the powerful object detection library [MMDetection](https://github.com/open-mmlab/mmdetection) while benefiting from all the features available on Determined's integrated deep learning platform.  In particular, get access to automated resource provisioning and experiment tracking with Determined along with state-of-the-art support for distributed training and hyperparameter tuning with little to no code modification.  
+
+## Capabilities
+Most of the configurations provided in the original [`mmdetection` repo](https://github.com/open-mmlab/mmdetection/tree/master/configs) are supported here.  You can see examples of how to submit an experiment to a Determined cluster with a particular config in [`maskrcnn.yaml`](maskrcnn.yaml).  The provided experiment configurations are as follows:
+* Mask R-CNN: [`maskrcnn.yaml`](mask_rcnn.yaml)
+* RetinaNet: [`retinanet.yaml`](retinanet.yaml)
+* Generalized Focal Loss: [`gfl.yaml`](gfl.yaml)
+
+## Performance
+We evaluated the performance of the library in Determined relative to the original mmdetection library both in terms of speed and predictive accuracy.  The comparisons were conducted on `n1-standard-64` instances with 8 Nvidia Tesla V100 GPUs.
+You can get the docker image we used by running
+```
+docker pull liamdetermined/development:mmdetection
```

Comment:
question: what's special about this docker image? Could we just put the libraries in a startup-hook? If not, maybe we should make an official determined docker images for examples.

Reply:
Setting up dependencies in the startup-hook takes too long.  I think it makes sense to consider having a docker repo for examples and the model hub.